### ISSUE TITLE
Move deploy settings to a separate dashboard

### ISSUE THREAD

**Issue by FransdeJong** [umbraco/Umbraco.Deploy.Issues#119]
I think it is time to move deploy settings away from the settings dashboard to it's own dashboard?
This way we can structure information in clear tabs. With the current implementation there is a lot going on on one screen especially when we want to make things more friendly to use like [this issue](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/118)

**Comment by tcmorris** [comment]
Agree, tidying up the display of the Deploy dashboard would be welcome. I think this could be done similar to how the Examine dashboard works however, whereby there is a link to view more details and then you can go back to the overview.
